src/similarity.rs

```diff
@@ -203,34 +210,35 @@ mod tests {
             .get_mut(&String::from("BFO:0000003"))
             .unwrap()
             .insert(String::from("BFO:0000004"));
-
+      
         let mut sco_predicate: HashSet<Predicate> = HashSet::new();
         sco_predicate.insert(String::from("subClassOf"));
-
+    
         let result = calculate_semantic_jaccard_similarity(
             &closure_map,
             "CARO:0000000",
             "BFO:0000002",
             &Some(sco_predicate.clone()),
         );
+    
         println!("{result}");
         assert_eq!(result, 2.0 / 3.0);
-
+        
+        // NO Predicate
+        // let mut sco_no_predicate: HashSet<Predicate> = HashSet::new();
         let result2 = calculate_semantic_jaccard_similarity(
             &closure_map,
             "BFO:0000002",
             "BFO:0000003",
             &Some(sco_predicate.clone()),
         );
         println!("{result2}");
-        assert_eq!(result2, 0.5);
-
+        assert_eq!(result2, 1.0 / 3.0);
```

Comment:
Why is this expected to be 0.333 when the similarity is between {"BFO:0000003", "BFO:0000002"} and {"BFO:0000003"} ?

Reply:
I believe this test is correct - the similarity is between {"BFO:0000003", "BFO:0000002"} and {"BFO:0000003", "BFO:0000004"}

NB that BFO:00000004 is included [here](https://github.com/monarch-initiative/semsimian/blob/a11f6438f9e7a0ae4d75b2d1e23cb0c81e4e33ac/src/test_utils.rs#L31)